fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -1340,6 +1342,81 @@ public void updateTableMeta(Database db, String tableName, Map<String, String> p
         }
     }
 
+
+    public void updateBinlogConfigMeta(Database db, Long tableId, Map<String, String> properties,
```

Comment:
We can log some information such as the config details for debug

Reply:
done